plugins/rbac-backend/src/service/role-manager.test.ts

```diff
@@ -1121,6 +1115,16 @@ describe('BackstageRoleManager', () => {
       expect(result).toBeTruthy();
     });
   });
+
+  describe('getRoles test', () => {
+    it('should returns 2 roles for user', async () => {
+      roleManager.addLink('user:default/test', 'role:default/rbac_admin');
+      const roles = await roleManager.getRoles('user:default/test');
+      expect(roles.length).toBe(2);
+      expect(roles[0]).toEqual('user:default/test');
+      expect(roles[1]).toEqual('role:default/rbac_admin');
+    });
```

Comment:
This follows up on my previous comment regarding `getRoles`. Here, we pass in a specific user to `getRoles` and expect the return of two roles. However, if we were to add additional links with `addLink`, it would retrieve all roles within Casbin, not just the two roles for the user.

Reproducible test
```
it('should returns 2 roles for user, additional links have been made', async () => {
      roleManager.addLink('user:default/test', 'role:default/rbac_admin');
      roleManager.addLink('user:default/test-two', 'role:default/rbac_admin');
      roleManager.addLink('user:default/test-three', 'role:default/rbac_admin_test_three');
      const roles = await roleManager.getRoles('user:default/test');
      expect(roles.length).toBe(2);
      expect(roles[0]).toEqual('user:default/test');
      expect(roles[1]).toEqual('role:default/rbac_admin');
    });
```
in this case we would receive 5 instead of 2.

Reply:
Thanks, I counted you proposal in the code and tests. But I think we should return only "role" string entity references without users for method getRoles.